be/Cf.Domain/Aggregates/Reviews/Review.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Cf.Domain.Aggregates.Jobs;
+
+namespace Cf.Domain.Aggregates.Reviews;
+
+public class Review : Entity
+{
+    public Guid JobId { get; set; }
+
+    public double Score { get; set; }
+
+    public string Description { get; set; }
+
+    public virtual Job Job { get; set; }
+
+    public Review() { }
```

Comment:
Is this needed?

Reply:
https://stackoverflow.com/a/31543487/15227176